README.md

```diff
@@ -148,6 +148,34 @@ endif()
 
 `thirdparty/nlohmann_json` is then a complete copy of this source tree.
 
+### Bazel
+
+In order to use this project in Bazel, the simplest approach is to download the
```

Comment:
Is such a long explanation required? The other build managers are only described briefly as the packages should follow the package manager's defaults.